src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculate {
```

Comment:
you've forgotten to implement getArea, re-read task requirement carefully and fix